c3.js

```diff
@@ -5097,6 +5097,7 @@ c3_chart_internal_fn.getDefaultConfig = function () {
         legend_padding: 0,
         legend_item_tile_width: 10,
         legend_item_tile_height: 10,
+        legend_format: undefined,
```

Comment:
Please, also add the documentation for this new option. The documentation site has been added to the repository.